File: Java/L03EX02/src/Vip.java

public class Vip extends Conta {
	
	public Vip(String nome, String nun_conta, Double saldo) {
		super(nome, nun_conta, saldo);
	
	}
	
	public Double getTarVip() {
		return (this.getSaldo() * 0.04);
	}
	
    public void mostraTarVip() {
    	System.out.println("Tarifa: R$ " + this.getTarVip());
    }
    
    
    public void mostraVip() {
    	System.out.println("Numero da Conta....: " + this.getNun_conta());
    	System.out.println("Nome...............: " + this.getNome());
    	System.out.println("Saldo..............: R$ " + this.getSaldo());
    	System.out.println("Tarifa VIP (4%)....: R$ " + this.getTarVip());
     	
    }

}

File: Swift/L03EX02/L03EX02/Vip.swift
// Vip.swift - L03EX02

import Foundation

public class Vip: Conta{
    
    
    public func getTarVip() -> Double{
        return (self.getSaldo() * 0.04)
    }
    
    public func printTarVip(){
        print("A tarifa VIP     (4%) do(a) \(self.getNome()): R$ \(self.getTarVip())\n")
    }
    
    
    public func mostraClienteVip(){
        print("Conta Número: \(self.getNumConta())")
        print("Nome Cliente: \(self.getNome())")
        print("saldo: R$ \(self.getSaldo())")
        print("A tarifa VIP (4%): R$ \(self.getTarVip())\n")
    }
    
}
